src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -154,6 +154,7 @@ export type Fiber = {
   // progressed work or if it is better to continue from the current state.
   progressedPriority: PriorityLevel,
   progressedWork: ProgressedWork,
+  newestWork: Fiber,
```

Comment:
Yeah maybe that's a bit better. Don't think either name makes it clear what it does or how it's different from `progressedWork`. I'll be sure we circle back to the naming before landing this PR.